inc/user.class.php

```diff
@@ -4031,7 +4031,8 @@ static function dropdown($options = []) {
       }
 
       // Check default value for dropdown : need to be a numeric
-      if ((strlen($p['value']) == 0) || (!is_numeric($p['value']) && $p['value'] != 'myself')) {
+      if ($p['value'] !== null
+          && ((strlen($p['value']) == 0) || !is_numeric($p['value']) && $p['value'] != 'myself')) {
```

Comment:
as implemented for groups : allow a null value. Needed when adding user_tech filter to have a unset value. If not, it shows the label 'myself' with the value 0 (which is invalid as the value should be 'myself')